sky/cli.py

```diff
@@ -3336,6 +3348,43 @@ def spot_queue(all: bool, refresh: bool, skip_finished: bool):
 _add_command_alias_to_group(spot, spot_queue, 'status', hidden=True)
 
 
+@spot.command('cost', cls=_DocumentedCodeCommand)
```

Comment:
Should we make this `cost-report` as well to be consistent with the `sky cost-report`?